spinegeneric/cli/generate_figure.py

```diff
@@ -280,8 +281,8 @@ def _get_flag(name):
         Get the flag of a country from the folder flags.
         :param name Name of the country
         """
-        path_flag = os.path.join(sg.__path__[0], 'flags', '{}.png'.format(name))
-        return plt.imread(path_flag)
+        with importlib.resources.path(spinegeneric.flags, f'{name}.png')) as path_flags:
```

Comment:
```suggestion
        with importlib.resources.path(spinegeneric.flags, f'{name}.png') as path_flags:
```

There is typo in doubled parentheses.

Reply:
thanks @valosekj, it is fixed in 0c69a46937ae04fc2697cf0e39b544196f2aadf4